Formula/countdown.rb

```diff
@@ -0,0 +1,17 @@
+class Countdown < Formula
+  desc "Terminla countdown timer"
+  homepage "https://github.com/antonmedv/countdown"
+  url "https://github.com/antonmedv/countdown/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "695bb6a57d7ba5a700ac16a7676132ee61c786a5e9a1ca774607d6f2ad5f39d7"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_equal "", pipe_output(bin/"countdown", "0m0s").strip
```

Comment:
Looks like there is nothing to assert, maybe just run the command itself
```suggestion
    system bin/"countdown", "0m0s"
```

Reply:
This is actually running the command itself. And tests parsing. 0s to make the test run faster.

Reply:
yeah, I understand the `0m0s` part, but I just dont understand the response assertion part. While the command has finished, why do we care about the `""` response?

Reply:
if `system bin/"countdown", "0m0s"` failed, the `test` step would also failed.

Reply:
Oh, make sense. Will drop it.

Reply:
unfortunately, we might have to switch to use your old test (it is due to the interactive nature of the command) 🤦

Reply:
Thanks for the try though.

Reply:
Tried with system and pty: both not working =( Will use pipe_output back.